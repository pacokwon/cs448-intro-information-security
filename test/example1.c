int main() {
  int x = 10;
  int y = 10 + x;
  int z = x - 10;
  int w = y / z;
  return 0;
}
