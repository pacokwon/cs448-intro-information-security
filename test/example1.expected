Potential Division-by-zero @ example1.c:main:7:12, %call = Top
Potential Division-by-zero @ example1.c:main:14:10, %call = Top
